ansible/rke2/airgap/playbooks/setup/setup-agent-nodes.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Setup RKE2 Agent Nodes
```

Comment:
Is this playbook's purpose just to add agent nodes to an existing setup? If that's the case, I wonder if this should just be a role instead.

Reply:
This can totally be refactored into a role. My oppions would be to either just remove this or add a todo to refactor later. Your assumption about the playbook is correct. If you add additional IPs to the airgap section of the inventory file, then this will join those extra agents to the cluster.

Reply:
we can refactor later